### PR TITLE
Reorganized file structure, updated to use KNP

### PR DESCRIPTION
**Pull request by esune**
Adds starter KNP for enabling backup jobs.

### CODE REVIEW

File: openshift/templates/backup/backup-deploy.yaml [hunk 1]
```diff
@@ -3,51 +3,33 @@ apiVersion: v1
...
-          - env=${TAG_NAME}
-          - $namespace=${ENVIRONMENT_NAME}
-        - - ext:name=${NAME}
+        Allow the ${NAME} to access databases in this namespace.
+      ingress:
+        - from:
+            - podSelector:
+                matchLabels:
+                  name: ${NAME}
+                  app: "backup-container"
```
Comment (WadeBarnes) [inline openshift/templates/backup/backup-deploy.yaml:22]: Should be a parameter

File: openshift/templates/backup/backup-deploy.yaml [hunk 2]
```diff
@@ -3,51 +3,33 @@ apiVersion: v1
...
+                  role: "backup-container"
```
Comment (WadeBarnes) [inline openshift/templates/backup/backup-deploy.yaml:23]: Should be a parameter

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
See comments

**Comment by AndrewJun97** [comment]
Hey @WadeBarnes , seems like the changes for the NetworkPolicies were proposed in this PR, though not merged yet.

**Comment by esune** [comment]
> 
> 
> Hey @WadeBarnes , seems like the changes for the NetworkPolicies were proposed in this PR, though not merged yet.

I missed this evidently, will sync-up with @WadeBarnes  and submit an update later today or tomorrow.

**Comment by esune** [comment]
@AndrewJun97 the PR has been updated, you can use the templates from my fork if you want to start working with the new templates before it is merged.

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**